ng/src/web/pages/help/about.js

```diff
@@ -0,0 +1,133 @@
+/* Greenbone Security Assistant
+*
+* Authors:
+* Steffen Waterkamp <steffen.waterkamp@greenbone.net>
+*
+* Copyright:
+* Copyright (C) 2017 Greenbone Networks GmbH
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public License
+* as published by the Free Software Foundation; either version 2
+* of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program; if not, write to the Free Software
+* Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React from 'react';
+import glamorous, {Col} from 'glamorous';
+
+import _ from 'gmp/locale.js';
+
+import ExternalLink from '../../components/link/externallink.js';
+import Img from '../../components/img/img.js';
+
+import Layout from '../../components/layout/layout.js';
+import Section from '../../components/section/section.js';
+
+const StyledLayout = glamorous(Layout) ({
+  margin: '0 auto',
+  maxWidth: '1100px',
+  display: 'flex',
```

Comment:
A Layout already has `display: flex`. Also it has properties to set wrap and the alignments. Just remove display, flexFlow and justifyContent.

Instead use it like
```jsx
<StyledLayout
  wrap
  align="center"
>
 ....
</StyledLayout>
```
in the render method at line 69.